source/includes/baremetal/devices.md

```diff
@@ -0,0 +1,690 @@
+      "password": "!rfl7qQ3p&Rc7A15vAyI",
+      "passwordReturnsUntil": 4,
+      "passwordExpires": "4",
+      "port": 22,
+      "user": "ubuntu"
+    },
+    "deviceIPs": {
+      "subnet": "91.191.214.64/29",
+      "netmask": "255.255.255.248",
+      "usableIps": [
+        "91.191.214.65",
+        "91.191.214.66",
+        "91.191.214.67",
+        "91.191.214.68",
+        "91.191.214.69",
+        "91.191.214.70"
+      ]
+    }
+  }
+}
+```
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/devices/:id</code>
+
+Retrieve a device in a given [environment](#administration-environments).
+
+| Attributes                                                   | &nbsp;                                                                                                                                   |
+| ------------------------------------------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------- |
+| `id`<br/>_string_                                            | The unique ID of the device.                                                                                                             |
+| `servicePlan`<br/>_string_                                   | The unique ID of the service associated with this device.                                                                                |
+| `name`<br/>_string_                                          | User given custom name.                                                                                                                  |
+| `deviceType` <br/>_string_                                   | Generic description of device. Usually type and rack unit size.                                                                          |
+| `primaryIp` <br/>_string_                                    | The first assigned public IP for accessing this device.                                                                                  |
+| `status` <br/>_string_                                       | active/inactive                                                                                                                          |
+| `monitorsTotal`<br>_int_                                     | Total # device monitors                                                                                                                  |
+| `monitorsUp`<br/>_int_                                       | Of passing device monitors.                                                                                                              |
+| `monitors`<br/>_string_                                      | monitors.                                                                                                                                |
```

Comment:
Could this description be improved?

Reply:
We don't have descriptions for these fields from the vendor.